src/main/handlers/helpers/transcriptionEngines/voskTranscribeFunction.ts

```diff
@@ -4,20 +4,28 @@ import fs from 'fs';
 import getVoskTranscript from '../../../../vosk';
 import { getAudioExtractPath } from '../../../util';
 import { TranscriptionFunction } from '../transcribeTypes';
+import getTranscriptionEngineConfig from '../../file/transcriptionConfig/getEngineConfig';
+import { LocalConfig, TranscriptionEngine } from '../../../../sharedTypes';
 
 interface VoskWord {
   end: number;
   start: number;
   word: string;
 }
 
+const getModelPath = async () => {
+  const localConfig = (await getTranscriptionEngineConfig(
+    TranscriptionEngine.VOSK
+  )) as LocalConfig;
+
+  if (localConfig.modelPath === null)
+    throw new Error('Vosk model path not configured');
+
+  return path.resolve(localConfig.modelPath);
```

Comment:
This is more of a suggestion. Could you create a custom error instead of throwing error? Similar to the ApiTokenError we have in the `assemblyAiTranscribeFunction.ts`. It'll make our error throwing/handling much more concise

```typescript
class ApiTokenError extends Error {}
```

Reply:
Good point. For readability, I have added two custom error types: `TranscriptionConfigError` and `LocalTranscriptionAssetNotFoundError`. Thanks! 😄